src/Components/PresentationalComponents/CVEPageDetailsDescription/TruncatedText.js

```diff
@@ -0,0 +1,49 @@
+import { Text, TextContent } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+
+const TruncatedText = props => {
+    const [isDescriptionTruncated, setDescriptionTruncated] = useState(true);
+    const [isDescriptionTooLong, setDescriptionTooLong] = useState(false);
+    const [shortDescription, setShortDescription] = useState('');
+
+    const { characterLimit, showMoreLabel, showLessLabel, content: description, component } = props;
```

Comment:
instead of assigning the value while you extract it: `content: description`, why don't you set it as props `description`

Reply:
and if you want to make it even more flexible you can extract those props like that 
`const TruncatedText = (characterLimit, showMoreLabel, showLessLabel, content: description, component) => {}`